src/sql/workbench/contrib/notebook/browser/cellViews/linkHandler.directive.ts

```diff
@@ -60,6 +61,15 @@ export class LinkHandlerDirective {
 		} catch {
 			// ignore
 		}
+		// Web mode linking to open files
+		if (isWeb) {
+			// only change scheme for file links (file links in web mode scheme are also http)
```

Comment:
How/why does a local link get turned into a web link in the first place? Would we not rather find the place that's being done and put this logic there to make the link use the vscode-remote scheme instead?

Reply:
The onclick target that we get back formats the href as such: `http://localhost:8000/filepath`. I have looked at where we save the links and output to the user (htmlMarkdownConverter and notebookMarkdown) but in each of these places, we are passing back the file path and not setting the uri. Since that is what we do here in linkHandler by creating the URI we get based on the href of the target. I believe it would be a better change here since it seems the target['href'] we get back is based on the window.location as well.

Reply:
Ok, so it's the web browser doing that then? Makes sense I guess.